src/dotnet/APIView/APIViewIntegrationTests/CosmosReviewRepositoryTests.cs

```diff
@@ -0,0 +1,222 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using APIViewWeb;
+using APIViewWeb.Models;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace APIViewIntegrationTests
+{
+    public class CosmosReviewRepositoryTests
+    {
+        private readonly CosmosClient _cosmosClient;
+        private CosmosReviewRepository _cosmosReviewRepository;
+        private readonly string _cosmosEmulatorConnectionString = "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
+
+        public CosmosReviewRepositoryTests()
+        {
+            _cosmosClient = new CosmosClient(_cosmosEmulatorConnectionString);
+            _cosmosReviewRepository = new CosmosReviewRepository(null, _cosmosClient);
+        }
+
+        [Theory]
+        [InlineData("Name", 64)]
+        [InlineData("Revisions[0].Files[0].Language", 2)]
+        public async Task GetReviewFirstLevelProprties_Test(string propertyName, int count)
+        {
+            await CreateTestDataIfNotExistAsync();
+            var property = await _cosmosReviewRepository.GetReviewFirstLevelProprtiesAsync(propertyName);
```

Comment:
seems like a more appropriate name will be `properties`

Reply:
Also the function has a typo in it: `GetReviewFirstLevelProprtiesAsync` `Proprties`